packages/api/src/libs/synthPrices.test.ts

```diff
@@ -0,0 +1,27 @@
+require("dotenv").config();
+import assert from "assert";
+import SynthPrices from "./synthPrices";
+import Web3 from "web3";
+
+// this is an integration test, dont run in ci
+describe.skip("synthPrices", function () {
```

Comment:
Is there a way to tag tests in mocha so that we can run them with a flag or something? Just curious. Otherwise, it'll get pretty tedious to try and run all the integration tests, so much so that it might make sense to isolate them in a particular location so we can run them with a particular command.

Reply:
good question, will have to look into that, but its a good idea to be able to run these independently

Reply:
ok so i've changed integration/e2e tests to be identified by filename, so `synthPrices.test.ts` is now `synthPrices.e2e.ts`.  `yarn test` will not run these, only files with `.test.ts`, `yarn e2e` will run any tests that are `.e2e.ts`.  the naming here is not the best, kind of lumping all non unit tests into `e2e`, but essentially tests which depend on external systems and could be non deterministic.